contracts/src/v0.1/Account.sol

```diff
@@ -23,11 +23,21 @@ contract Account is IAccount, ITypeAndVersion {
     address private sRequestedOwner; // For safely transferring acc ownership
     uint256 private sBalance; // Common $KLAY balance used for all consumer requests
     uint64 private sReqCount; // For fee tiers
+    AccountType private sAccountType;
+    uint256 private sStartTime; // activated time use for
```

Comment:
I think the comments do not necessarily help with understanding. I will give my suggestions below and feel free to use them or not.

```Solidity
    uint256 private sStartTime; // subscription activation time
    uint256 private sPeriod; // subscription period
    uint256 private sPeriodReqCount; // number of allowed requests within a subscription period
    uint256 private sServiceFeeRatio; // discounted service fee denominated in basis points (e.g. 9500 -> 5 % discount)
    uint256 sSubscriptionPrice; // only for KLAY_SUBSCRIPTION
```